jekyll/_includes/no-script.html

```diff
@@ -0,0 +1,3 @@
+<noscript>
+    <style>.row,.control{display:initial}.loading-deferred{display:block}#full-height{visibility:visible}</style>
```

Comment:
is this to ensure that these styles are set in the event that scripting is turned off in the browser?

Reply:
Also as to why we use display none.

It helps with regards to CLS and generally avoiding jankiness until the experiment is loaded.

Reply:
Let's add a comment about your great explanation in the file, please 🙏